scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -213,7 +226,12 @@ function doImportProject() {
 
   local setup_templ="${setup_templ_dir}/${setup_templ_name}"
   local setup_ws="${setup_ws_dir}/${setup_ws_name}"
-  
+
+  if doIsWindows # cut drive letter from Windows path only (not neccesarry under macOS/Linux)
+  then
+    proj_dir=${proj_dir:2} # for example 'C:/Users/' becomes '/Users/'
```

Comment:
never do things like this. Use https://cygwin.com/cygwin-ug-net/cygpath.html instead (also available in git-bash/MINGW/MSYS).

Reply:
Ok, @markusschuh already gave me a hint in that direction but I'm not sure how to get rid of the leading drive letter that way. His suggested code was:

if command -v cygpath >/dev/null
then
  proj_dir=$( cygpath --mixed "${proj_dir}" )
fi